### PR TITLE
Shorten namespace to Interop\Async and correct PHP version requirement

### PR DESCRIPTION
**Pull request by kelunik**
PR for #34.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
     "name": "async-interop/event-loop",
-    "description": "An event loop interface",
+    "description": "An event loop interface for interoperability.",
```
Comment (bwoebi) [inline composer.json]: Interoperability is a _goal_. The project itself is just the interface.

Comment (kelunik) [inline composer.json]: That's why there's a _for_. The goal should be in the description.

Comment (bwoebi) [inline composer.json]: I still feel a bit uneasy with that. May you please split that into another PR so that others may agree or disagree on it separately?

Comment (kelunik) [inline composer.json]: Well, even the package vendor already has `interop` in it.

Comment (kelunik) [inline composer.json]: Moved to #38.
